pkger/models.go

```diff
@@ -547,6 +562,8 @@ func (c chart) validProperties() []failure {
 		fails = append(fails, c.Colors.hasTypes(colorTypeScale)...)
 		fails = append(fails, validGeometry(c.Geom)...)
 		fails = append(fails, c.Axes.hasAxes("x", "y")...)
+	case ChartKindGauge:
+		fails = append(fails, c.Colors.hasTypes(colorTypeMin, colorTypeThreshold, colorTypeMax)...)
```

Comment:
are all these colors required for the chart to showup? or are some optional and we have graceful fallbacks? if they are required, lets keep it this as is 👌

Reply:
they're all required